manly.py

```diff
@@ -136,14 +110,36 @@ def manly(command):
     return title, output
 
 
+USAGE_EXAMPLE = """example:
+    $ manly rm --preserve-root -rf
+
+    rm - remove files or directories
+    ================================
+
+        -f, --force
+                ignore nonexistent files and arguments, never prompt
+
+        --preserve-root
+                do not remove '/' (default)
+
+        -r, -R, --recursive
+                remove directories and their contents recursively"""
+
+VERSION = (
+    "manly %s\nCopyright (c) 2017 %s.\nMIT License: see LICENSE.\n\n"
+    "Written by %s and Mark Jameson."
+) % (__version__, __author__, __author__)
```

Comment:
I guess I could, but this is more since next PR is going to involve splitting everything into seperate files, so I was pooling everything into its seperate parts ready for the split

Reply:
Can you give an example of this split? I like it in one file.